### PR TITLE
feat: confirmation plugin

### PR DESCRIPTION
**Pull request by jacoobes**

### CODE REVIEW

File: TypeScript/confirmation.ts [hunk 1]
```diff
@@ -0,0 +1,138 @@
+//@ts-nocheck
+/**
+ * @author @HighArcs [<@504698587221852172>]
+ * @version 1.0.0
+ * @description asks the user for a confirmation message before executing the command
```
Comment (EvolutionX-10) [inline TypeScript/confirmation.ts]: description should not be written like this, please refer other plugin files for correct examples

File: TypeScript/confirmation.ts [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+ * @license null
```
Comment (EvolutionX-10) [inline TypeScript/confirmation.ts]: no need for license line

**Review by EvolutionX-10 — CHANGES_REQUESTED** [review]
i assume this has been tested properly on a test bot. please address the comments

**Comment by jacoobes** [comment]
Yes

File: TypeScript/confirmation.ts [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+ * import { CommandType, commandModule } from "@sern/handler";
+ *
+ * // YOU MUST HAVE GUILD MESSAGE REACTION INTENTS ON YOUR CLIENT!!!!
+ * export default commandModule({
+ *  type : CommandType.Both
+ *  plugins: [confirmation()],
+ * 	execute: (ctx, args) => {
+ * 		ctx.reply('hola')
+ * 	}
+ * })
```
Comment (EvolutionX-10) [inline TypeScript/confirmation.ts:19]: bro forgot to close the code block 💀

**Review by EvolutionX-10 — CHANGES_REQUESTED** [review]
xD

File: TypeScript/confirmation.ts [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+ * // YOU MUST HAVE GUILD MESSAGE REACTION INTENTS ON YOUR CLIENT!!!!
+ * export default commandModule({
+ *  type : CommandType.Both
+ *  plugins: [confirmation()],
+ * 	execute: (ctx, args) => {
+ * 		ctx.reply('hola');
+ * 	}
+ * })
+ * ```
+ **/
```
Comment (EvolutionX-10) [inline TypeScript/confirmation.ts]: I swear you're gonna be so angry on me right now
```suggestion
 */
```

**Review by EvolutionX-10 — CHANGES_REQUESTED** [review]

File: TypeScript/confirmation.ts (hunk 4 above)
Comment (jacoobes) [reply TypeScript/confirmation.ts]: 😟I'll fix one sec

**Review by EvolutionX-10 — APPROVED** [review]

**Pull request merged**